testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -370,35 +370,6 @@ Feature: PXE boot a Retail terminal
     And I am on the Systems page
     Then I should see the terminals imported from the configuration file
 
-@proxy
```

Comment:
Please test this carefully. I think there was some specific reason to have this "cheating" part twice (one for pxeboot part, once for mass import part) ;-) I can't remember exactly, it is some time ago.

Reply:
Yes, I'm currently testing it (meaning i have to rebuild the image with kiwi :-( )

My assumption was that you were "cheating" twice because this feature is the result of merging of two features (the pxe boot, and the mass import). So we were preparing things twice, and this is how we ended up with this duplicated.

Of course, I might be wrong.

Reply:
> My assumption was that you were "cheating" twice because this feature is the result of merging of two features (the pxe boot, and the mass import). So we were preparing things twice, and this is how we ended up with this duplicated.

That could be it. At least I hope so :-)

Reply:
Hmmm, it looks like you are right. After `Apply the highstate to take into account the imported formulas` the bind formula is kind of reset (not really sure why).

Reply:
You were right. That is made necessary by mass import script that seems to fiddle around bind records.

Resurrected.

Reply:
> Tests passed with success.

Perfect :-) I am approving now...